extract_verbs.sh

```diff
@@ -114,7 +114,14 @@ find extracted -type f -exec cat {} \; > all-extracted.txt
 # Extract verbs with TreeTagger
 # N.B. treetagger segfaults with the single big file, run it over each article instead
 #cat all-extracted.txt | treetagger/cmd/tree-tagger-italian | grep VER | sort -u > verbi.txt
-find extracted -type f -exec bash -c "cat '{}' | treetagger/cmd/tree-tagger-"$LANGUAGE" | grep VER >> verbs" \;
+#For english corpus
+if [$LANGCODE == 'en']; then
```

Comment:
I think we can make this clearer by moving the `verbTag` variable up in the switch statement which handles all the languages.